api/metric/counter.go

```diff
@@ -30,41 +30,41 @@ type Int64Counter struct {
 	commonMetric
 }
 
-// Float64CounterHandle is a handle for Float64Counter.
+// Float64CounterBoundInstrument is a boundInstrument for Float64Counter.
 //
-// It inherits the Release function from commonHandle.
-type Float64CounterHandle struct {
-	commonHandle
+// It inherits the Release function from commonBoundInstrument.
+type Float64CounterBoundInstrument struct {
+	commonBoundInstrument
 }
 
-// Int64CounterHandle is a handle for Int64Counter.
+// Int64CounterBoundInstrument is a boundInstrument for Int64Counter.
 //
-// It inherits the Release function from commonHandle.
-type Int64CounterHandle struct {
-	commonHandle
+// It inherits the Release function from commonBoundInstrument.
+type Int64CounterBoundInstrument struct {
+	commonBoundInstrument
 }
 
-// AcquireHandle creates a handle for this counter. The labels should
+// AcquireBoundInstrument creates a boundInstrument for this counter. The labels should
 // contain the keys and values for each key specified in the counter
 // with the WithKeys option.
 //
 // If the labels do not contain a value for the key specified in the
 // counter with the WithKeys option, then the missing value will be
 // treated as unspecified.
-func (c *Float64Counter) AcquireHandle(labels LabelSet) (h Float64CounterHandle) {
-	h.commonHandle = c.acquireCommonHandle(labels)
+func (c *Float64Counter) AcquireBoundInstrument(labels LabelSet) (h Float64CounterBoundInstrument) {
+	h.commonBoundInstrument = c.acquireCommonBoundInstrument(labels)
 	return
 }
 
-// AcquireHandle creates a handle for this counter. The labels should
+// AcquireBoundInstrument creates a boundInstrument for this counter. The labels should
 // contain the keys and values for each key specified in the counter
 // with the WithKeys option.
 //
 // If the labels do not contain a value for the key specified in the
 // counter with the WithKeys option, then the missing value will be
 // treated as unspecified.
-func (c *Int64Counter) AcquireHandle(labels LabelSet) (h Int64CounterHandle) {
-	h.commonHandle = c.acquireCommonHandle(labels)
+func (c *Int64Counter) AcquireBoundInstrument(labels LabelSet) (h Int64CounterBoundInstrument) {
```

Comment:
@jmacd (asking you since you opened #390) in the spec this method was originally named `GetHandle`, not `AcquireHandle`, and has now been renamed to `Bind`. Should I also rename to follow the spec?

Reply:
I'll suggest just replace AcquireHandle with Bind

Reply:
Yes. The Go code here has diverged from the spec more than once, we should replace AquireHandle with Bind.